libs/admin/src/lib/services/admin.service.ts

```diff
@@ -55,10 +65,18 @@ export class AdminService {
     return this._httpClient.get('/api/admin/user').pipe(map((res) => res as User[]));
   }
 
+  public getUserNew(id: string): Observable<User> {
+    return this._httpClient.get(`/api/users/${id}`).pipe(map((res) => res as User));
+  }
+
   public getWorkgroups(): Observable<Workgroup[]> {
     return this._httpClient.get('/api/workgroups').pipe(map((res) => res as Workgroup[]));
   }
 
+  public getWorkgroup(id: string): Observable<Workgroup> {
+    return this._httpClient.get(`/api/workgroups/${id}`).pipe(map((res) => res as Workgroup));
```

Comment:
Mapping responses just to cast them is not necessary - we can just specify the type in the request:
```suggestion
    return this._httpClient.get<Workgroup>(`/api/workgroups/${id}`);
```
Note that type casts are a no-op when TS is compiled to JS - `a as B` is compiled to just `a`.